src/routes/auth-token/create.js

```diff
@@ -1,13 +1,13 @@
 import Joi from 'joi'
 import jwt from 'jsonwebtoken'
-import ethUtil from 'ethereumjs-util'
+import ethSigUtil from 'eth-sig-util'
 import RestifyErrors from 'restify-errors'
 
 import models from '../../models'
 import config from '../../config'
 import logger from '../../services/logger'
 
-const hashedPersonalMessage = ethUtil.hashPersonalMessage(ethUtil.toBuffer(config.personalMessageToSign))
+const { typedDataToSign } = config
```

Comment:
Remove this line and just use `config.typedDataToSign` below, i.e.:

```javascript
const recoveredAddress = ethSigUtil.recoverTypedSignature({
  sig: request.parameters.signedData,
  data: config.typedDataToSign,
})
```

Reply:
I'm doing something similar on the front end and it complained that I wasn't destructuring. Happy to make this change though.